scalability_and_performance/recommended-host-practices.adoc

```diff
@@ -17,6 +17,8 @@ include::modules/master-node-sizing.adoc[leveloffset=+1]
 
 include::modules/recommended-etcd-practices.adoc[leveloffset=+1]
 
+include::modules/etcd-defrag.adoc[leveloffset=+1]
```

Comment:
You must also go into the modules/etcd-defrag.adoc and note in the file's metadata that the module is included in scalability_and_performance/recommended-host-practices.adoc

Reply:
Done.